server/viame_server/serializers/viame.py

```diff
@@ -3,11 +3,49 @@
 """
 import csv
 import re
+from dataclasses import dataclass, asdict, field
+from typing import List, Dict, Tuple, Optional, Union, Any
 
 from girder.models.file import File
 
 
-def _deduceType(value):
+@dataclass
+class Feature:
+    """Feature represents a single detection in a track."""
+
+    frame: int = None
+    bounds: List[float] = None
+    head: Optional[Tuple[str, str]] = None
+    tail: Optional[Tuple[str, str]] = None
+    fishLength: Optional[float] = None
+    attributes: Optional[Dict[str, Union[bool, float, str]]] = None
+
+
+@dataclass
+class Track:
+    begin: int
+    end: int
+    key: int
+    features: List[Feature] = field(default_factory=lambda: [])
+    confidencePairs: Dict[str, float] = field(default_factory=lambda: {})
```

Comment:
I found out while working on the client that it's also nicer to keep confidencePairs as a list of lists:

```python
[
  ['name', 0.86],
  ['v2', 1],
]
```

Reply:
I'm in favor of that, however that means we'd have to remove this section of code:
https://github.com/VIAME/VIAME-Web/blob/91b3c83b6d89bfa85e1ebb032777ab9b997efd98/server/viame_server/serializers/viame.py#L171-L174

Is there any downside to doing this? I'm not sure what that comment means.